src/software/jetson_nano/services/power.cpp

```diff
@@ -5,25 +5,27 @@
 #include <cstdint>
 
 #include "proto/power_frame_msg.nanopb.h"
+#include "software/jetson_nano/services/network/power_service_exception.h"
 
 PowerService::PowerService()
 {
     if (!boost::filesystem::exists(DEVICE_SERIAL_PORT))
     {
-        LOG(FATAL) << "PLUG THE USB INTO THE JETSON NANO";
+        throw PowerServiceException("USB not plugged into the Jetson Nano");
```

Comment:
can we make this a runtimeexception instead or some other generic exception

Reply:
Sounds good